components/Dialogs/src/MDCAlertControllerView.h

```diff
@@ -67,4 +67,64 @@
  */
 @property(nonatomic, assign) BOOL enableRippleBehavior;
 
+#pragma mark - Adjustable Insets
+
+/**
+ The edge insets around the title icon or title icon view against the dialog edges (top, leading,
+ trailing) and the title (bottom). Note that `titleIconInsets.bottom` takes precedence over
+ `titleInsets.top`.
```

Comment:
What does it mean for "`titleIconInsets.bottom` [to take] precedence over `titleInsets.top`"? If `titleIconInsets.bottom` == 10 and `titleInsets.top` is 20, does this mean the spacing would be 10?

If so, this is surprising behavior. I'd expect this behavior to work by taking the max of both values, similar to how the html box model handles [margin collapsing](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing). If this is not how this behavior is presently working please adjust it prior to publishing this API so that we do not have to introduce a behavioral flag.

This principle should apply throughout to any other insets that might overlap.